fiona/__init__.py

```diff
@@ -202,6 +202,16 @@ def open(fp, mode='r', driver=None, schema=None, crs=None, encoding=None,
     crs_wkt : str
         An optional WKT representation of a coordinate reference
         system.
+    ignore_fields : list
+        List of field names to ignore on load.
+    ignore_geometry : bool
+        Ignore the geometry on load.
+    include_fields : list
+        List of a subset of field names to include on load.
+    wkt_version : str
+        WKT Version to use to for the CRS.
+        (WKT1_GDAL, WKT1_ESRI, WKT2_2015, WKT2_2018)
+        Only works with GDAL 3+. Default is WKT1_GDAL.
```

Comment:
Version GDAL 3+ is now irrelevant.
```suggestion
        Default is WKT1_GDAL.
```
Note the implementation here has a default of None, where FORMAT is not passed as an option. Does GDAL assume the default WKT1_GDAL? ([ref](https://gdal.org/api/ogrspatialref.html#_CPPv4NK19OGRSpatialReference11exportToWktEPPcPPCKc))

Reply:
It is the default until GDAL 4 (https://github.com/OSGeo/gdal/issues/4681).

Reply:
I updated it to hopefully be more clear.